src/renderer/pages/PreferencesPage/index.tsx

```diff
@@ -14,6 +14,12 @@ const PreferencesDiv = styled.div`
   ${styles.meat};
 `;
 
+const expTextColor = global.ReduxStore.getState().preferences.lightMode
```

Comment:
If memory serves, this is attached to explanatory text in the preferences menu.  I'll have to get to the other comments and suggestions as the week goes, just happy to hear back about this!

Reply:
@3ter Also seeking input here if you have any.  The hook for this component is set to run only when the tab is open rather than when anything in global rootstate is changed, so when you click the checkbox to go from dark into light the text in the preferences menu itself can be hard to read until the preferences menu is reopened.  Any recommendations to deal with this?
![Screenshot from 2024-03-06 14-05-12](https://github.com/itchio/itch/assets/12357690/a1a65c41-ad56-4da2-a06f-4c096ba7d708)